cli/api/utils.go

```diff
@@ -41,11 +44,19 @@ func GetAgentIP(defaultRPCPort int) string {
 	if options.Endpoint != "" {
 		return options.Endpoint
 	}
-	agentIP, err := utils.GetIPAddress()
-	if err != nil {
-		panic(err)
+
+	var agentIP string
+	var err error
+	for i := 1; i <= networkAccessAttempts; i++ {
+		if agentIP, err = utils.GetIPAddress(); err == nil {
```

Comment:
If SERVICED_OUTBOUND_IP is set, then options.Endpoint will not be empty and offline install will work, right? Or is SERVICED_OUTBOUND_IP for something else?